src/etc/platform-intrinsics/powerpc.json

```diff
@@ -72,6 +72,55 @@
             "llvm": "vmin{0.kind}{0.data_type_short}",
             "ret": "i(8-32)",
             "args": ["0", "0"]
+        },
+        {
+            "intrinsic": "sub{0.kind}{0.data_type_short}s",
+            "width": [128],
+            "llvm": "vsub{0.kind}{0.data_type_short}s",
+            "ret": "i(8-32)",
+            "args": ["0", "0"]
+        },
+        {
+            "intrinsic": "subc",
+            "width": [128],
+            "llvm": "vsubcuw",
+            "ret": "u32",
+            "args": ["0", "0"]
+        },
```

Comment:
The underlying instruction for both signed and unsigned is `vsubcuw`.

I'm currently exposing the altivec intrinsics, then I'll add the VSX and POWER8-only ones.
The safe altivec.h-like-api would live in a separate crate.

Please refer to [altivec.h](https://github.com/llvm-mirror/clang/blob/master/lib/Headers/altivec.h) to see what I'm miming :)